tests/complex/fake_sam.py

```diff
@@ -24,15 +50,17 @@ def do_GET(self):
             }
             self.write_json(content)
         else:
-            content = {"result": "ok",
-            }
+            content = {"result": "ok"}
             self.write_json(content)
 
     def do_POST(self):
-        print("[FakeSam] POST", self.path)
+        """
+        Handler for POST method
+        """
+        print("[FakeSam] POST", self.path, self.server.code)
         if self.server.code:
             self.write_json({}, status_code=self.server.code, headers={"Retry-After": "60"})
-        elif self.path.startswith('/hypervisors'):
+        elif self.path.startswith('/hypervisors') or self.path.startswith('//hypervisors'):
```

Comment:
Why would it have an extra "/"? Best I could guess is it assumes that there is a hander defined when there isn't one (like it should be set to "candlepin" resulting in "/candlepin/hypervisors".

Reply:
I don't know